packages/renderer/src/components/atoms/Currency/Currency.tsx

```diff
@@ -15,15 +15,19 @@
 // =============================================================================
 
 import { useMemo } from "react";
+import type { FormatNumberOptions } from "react-intl";
 import { useIntl } from "react-intl";
 
-interface CurrencyProps {
+type CurrencyFormatType = "symbol" | "code" | "name" | "narrowSymbol";
+
+interface CurrencyProps extends FormatNumberOptions {
   currencyCode?: string;
+  currentDisplay?: CurrencyFormatType;
```

Comment:
`currentDisplay` isn't a good choice for a name. How about `format`?

Reply:
I don't know how did that. 🤦 🤦